valohai_yaml/pipelines/conversion.py

```diff
@@ -55,21 +54,25 @@ def convert_deployment_node(self, node: DeploymentNode) -> ConvertedObject:
     def convert_executionlike_node(self, node: Union[ExecutionNode, TaskNode]) -> ConvertedObject:
         node_data = node.serialize()
         step_name = node_data.pop("step")
-        override = node_data.pop("override", {})
         step = self.config.get_step_by(name=step_name)
         if not step:  # pragma: no cover
             raise ValueError(f"Step {step_name} not found in {self.config}")
+
         step_data = step.serialize()
-        step_data.update(override)
+        node_data.pop("override", {})
```

Comment:
Won't this now drop everything else that could have been overridden? (`image`, `command`, ..?)

Reply:
Yeah, my bad. Will fix asap